### PR TITLE
add a Feature list

### PR DESCRIPTION
**Pull request by Simon-Laux**
A list of the features every DC Client should have as reference for developers that develop Clients.

Please share your opinions about this list and feel free to add thinks I forgot(the feature section is the most interesting right now, the `feature request` list is far from complete).

### CODE REVIEW

File: Features.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+Last update: 
+05/22/2019 @ 11:15pm (UTC)
+
+## Features
+What the clients have to implement to be feature complete. This can act as a checklist when implementing a new client.
+*Only those which are already possible with the current core.*
+  
+- Searching
+  - For Chats
+  - For Messages by content
```
Comment (adbenitez) [inline Features.md]: and searching contacts?

File: Features.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+  - Text
+    - Emoji Picker
+  - Attachment
+    - Video
+    - Image
+    - Audio
+    - File
+  - Location Attribute
+  - Drafts
+  - Star Message
```
Comment (cyBerta) [inline Features.md]: > *Only those which are already possible with the current core.*

starring messages are not implemented yet on ios/android and afaik in the core.

Comment (Simon-Laux) [reply Features.md]: you're right that is only halfway implemented

**Comment by Simon-Laux** [comment]
We should redo this if we want it

**Comment by Simon-Laux** [comment]
when/if we redo this we can link https://github.com/deltachat/interface/pull/42

**Pull request closed without merging**